app/src/main/java/org/thoughtcrime/securesms/home/HomeActivity.kt

```diff
@@ -640,31 +644,29 @@ class HomeActivity : PassphraseRequiredActionBarActivity(),
                 lifecycleScope.launch(Dispatchers.Main) {
                     val context = this@HomeActivity
                     // Cancel any outstanding jobs
-                    DatabaseComponent.get(context).sessionJobDatabase().cancelPendingMessageSendJobs(threadID)
-                    // Send a leave group message if this is an active closed group
-                    if (recipient.address.isClosedGroup && DatabaseComponent.get(context).groupDatabase().isActive(recipient.address.toGroupString())) {
-                        try {
-                            GroupUtil.doubleDecodeGroupID(recipient.address.toString()).toHexString()
-                                .takeIf(DatabaseComponent.get(context).lokiAPIDatabase()::isClosedGroup)
-                                ?.let { MessageSender.explicitLeave(it, false) }
-                        } catch (ioe: IOException) {
-                            Log.w(TAG, "Got an IOException while sending leave group message")
-                        }
-                    }
+                    sessionJobDatabase
+                        .cancelPendingMessageSendJobs(threadID)
+
                     // Delete the conversation
-                    val v2OpenGroup = DatabaseComponent.get(context).lokiThreadDatabase().getOpenGroupChat(threadID)
+                    val v2OpenGroup = lokiThreadDatabase
```

Comment:
v2OpenGroup is a misleading name, could be a good opportunity to rename this with community phrasing.

Reply:
Changed. I've refrained from changing the `getOpenGroupChat` method and `OpenGroupManager` here though, as they are way too many changes to other places.

Reply:
Will leave the bigger change for later